src/demo/components/login/login.component.html

```diff
@@ -106,6 +106,7 @@ <h1>Demo</h1>
         id="publicLogin"
         mat-flat-button
         color="primary"
+        [disabled]="!(loginForm.controls.environment.value === 'demo')"
```

Comment:
What's this?

Reply:
The stop people from attempting to login the public customer in an environment other than demo. I've fixed this an alternative way now.